buildpack.go

```diff
@@ -35,8 +35,8 @@ func (bp Buildpack) noHomepage() Buildpack {
 	return bp
 }
 
-func (bp Buildpack) Lookup(buildpacksDir string) (*BuildpackTOML, error) {
-	bpTOML := BuildpackTOML{}
+func (bp Buildpack) Lookup(buildpacksDir string) (*DefaultBuildpackTOML, error) {
```

Comment:
As I'm looking into this, it's slightly more challenging than expected. For build, it's sufficient that the `BuildpackStore.Lookup` method returns something that satisfies the `Buildpack` interface i.e., it can `Build()`. For detect, the thing returned by `Lookup` also needs to have `API`, `Buildpack`, `Order`, etc. Should I add those to the interface? Or is there a cleaner way to do this that I'm not thinking of?

Reply:
let's punt this for now